channeldb/graph.go

```diff
@@ -2143,7 +2149,7 @@ func (c *ChannelGraph) FilterKnownChanIDs(chansInfo []ChannelUpdateInfo,
 				// and we let it be added to the set of IDs to
 				// query our peer for.
 				case isZombie && !isStillZombie:
-					err := c.markEdgeLive(tx, scid)
+					err := c.markEdgeLiveUnsafe(tx, scid)
```

Comment:
I like this concept could we also change the `delChannelEdge` to `delChannelEdgeUnsafe` while we fix this one ?

Reply:
good idea! will add a commit